pkg/operator/api/userconfig/apis.go

```diff
@@ -54,6 +56,19 @@ var apiValidation = &cr.StructValidation{
 				AllowCortexResources: true,
 			},
 		},
+		{
+			StructField:         "Requesthandler",
+			StringPtrValidation: &cr.StringPtrValidation{},
+		},
+		{
+			StructField: "ModelType",
+			StringValidation: &cr.StringValidation{
+				AllowedValues: ModelTypeStrings(),
```

Comment:
Do we need `Required: true`? I guess it would be blocked by `AllowEmpty` being `false`, but it might be a better error message if we also specify `Required: true`, can you try it out?